doc/classes/Input.xml

```diff
@@ -180,6 +180,7 @@
 			<argument index="0" name="action" type="String">
 			</argument>
 			<description>
+				Returns [code]true[/code] once if the action event was pressed.
```

Comment:
I don't think "once" makes sense. All functions only return a value once, each time you call them.

    Returns [code]true[/code] if the action event state just changed to pressed.